templates/package-detail.html

```diff
@@ -1,55 +1,35 @@
 {% extends "base.html" %}
 {% import "macros.html" as macros %}
-{% block title %}Koschei - {{ base.name }}{% endblock %}
-{% macro change_link(change) %}
-{% if change %}
-<td>
-    <a href="{{ url_for(
-        'affected_by',
-        dep_name=change.dep_name,
-        epoch1=change.prev_epoch or '0',
-        version1=change.prev_version,
-        release1=change.prev_release,
-        epoch2=change.curr_epoch or '0',
-        version2=change.curr_version,
-        release2=change.curr_release,
-        collection=collection.name,
-        ) }}">
-        other affected
+
+{% macro collapse_btn(items, max, type, id) %}
+{% if items|length > max %}
+<div class="row">
+  <div class="col-sm-2 offset-5">
+    <a class="btn btn-secondary btn-sm kk-toggle-button"
+       data-toggle="collapse" href="#kkid-{{ type }}-{{ id }}" aria-expanded="false">
+      <span class="kk-collapsed">
+        <i class="fa fa-angle-double-down"></i>
+        show {{ items|length - max }} more
+        <i class="fa fa-angle-double-down"></i>
+      </span>
+      <span class="kk-expanded">
+        <i class="fa fa-angle-double-up"></i>
+        collapse
+        <i class="fa fa-angle-double-up"></i>
+      </span>
     </a>
-</td>
-{% else %}
-<td></td>
+  </div>
+</div>
 {% endif %}
 {% endmacro %}
-{% macro next_change(changes) %}
-{% set change = inext(changes, None) %}
-{{ change | format_depchange | columnize(css_class="mono") }}
-{{ change_link(change) }}
-{% endmacro %}
-{% block content %}
-<div class="pageHeader">{{ base.name }}</div>
 
+{% block title %}Koschei - {{ base.name }}{% endblock %}
+{% block subheader %}
+<h2>{{ base.name }}</h2>
 {{ macros.collection_chooser(allow_all=False) }}
+{% endblock %}
 
-<div class="package-collection-chooser {% if collection %}float-right{% endif %}">
-    {% if collection %}
-    <h4>State in all collections</h4>
-    {% else %}
-    <h4>Choose a collection</h4>
-    {% endif %}
-    <ul>
-        {% for (collection, package) in all_packages  %}
-        <li>
-            <a href="{{ url_for('package_detail', name=package.name) }}{{
-                page_args(clear=True, collection=collection.name) }}">
-                {{ package.state_icon }} {{ collection }}
-            </a>
-        </li>
-        {% endfor %}
-    </ul>
-</div>
```

Comment:
And with the current tabs, if you didn't select a collection, none of them is displayed as active, so you don't know which collection you're looking at.